migrations/sql/R__stored_procedure_for_mine_information.sql

```diff
@@ -199,6 +200,7 @@ CREATE OR REPLACE FUNCTION transfer_mine_information() RETURNS void AS $$
             CREATE TEMP TABLE IF NOT EXISTS pmt_now (
                 lat_dec   numeric(11,7),
                 lon_dec   numeric(11,7),
+                mine_location_description    varchar,
```

Comment:
Character count on `varchar`?

Reply:
the description is a free range text input where people put notes on how to get to a mine. I didnt see any business requirement on length so I left it open since varchar will only take as much space as needed and nothing more

Reply:
Works for me. Marking as resolved